README.md

```diff
@@ -5,15 +5,14 @@
 
 # Odyssey Design System
 
-This repository contains the code for Okta's Design System Odyssey. It includes Icons, SCSS, and React Components used to build products and experiences at Okta.
+This repository contains the code for Okta's Design System Odyssey. It includes Icons and React Components used to build products and experiences at Okta.
 
 ## Packages
 
-| Package/README                                                                                      | Description                                                        |
-| --------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ |
-| [@okta/odyssey-icons](https://github.com/okta/odyssey/tree/master/packages/odyssey-icons/README.md) | Odyssey SVG icons.                                                 |
-| [@okta/odyssey-react](https://github.com/okta/odyssey/tree/master/packages/odyssey-react/README.md) | Odyssey components for [React](https://reactjs.org/).              |
-| [@okta/odyssey-scss](https://github.com/okta/odyssey/tree/master/packages/odyssey-scss/README.md)   | Odyssey SCSS, includes components, includes base and reset styles. |
+| Package/README                                                                                      | Description                                           |
+| --------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| [@okta/odyssey-icons](https://github.com/okta/odyssey/tree/master/packages/odyssey-icons/README.md) | Odyssey SVG icons.                                    |
+| [@okta/odyssey-react](https://github.com/okta/odyssey/tree/master/packages/odyssey-react/README.md) | Odyssey components for [React](https://reactjs.org/). |
```

Comment:
This list should really focus on the packages that will be published to public NPM. As of now:

`@okta/odyssey-design-tokens`
`@okta/odyssey-react-theme`
`@okta/odyssey-react`

Reply:
Good feedback. However, we do not support IE11 at this time. Should I also add that in the README?

Reply:
hmmmm good thought. I'm hopeful we'll be working on that quickly first thing in February and resolve any issues but in an effort to be brutally honest we probably should clarify that in the top level README **and** each of the README's that make mention of browser support: `@okta/odyssey-react-theme` and @okta/odyssey-react`

Reply:
This looks like it still needs to be updated to show the correct packages cc @arnoldsandoval-okta